harvard-lts/jstor_aspace_transformer
# remove https and self signed cert

**remove https and self signed cert.**
* * *

**JIRA Ticket**: [(link)](https://at-harvard.atlassian.net/browse/LTSK8S-494)

# What does this Pull Request do?
 Remove https ref in tasks and self signed cert in supervisord files

# How should this be tested?

Check out this branch and confirm docker comes up cleanly (done)
Deploy to dev and confirm itest passes (done)

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? N
- integration tests? N

# Interested parties
Tag (@ mention) interested parties
